src/common/json_output.rs

```diff
@@ -137,3 +143,72 @@ impl EnclaveBuildInfo {
         EnclaveBuildInfo { measurements }
     }
 }
+
+/// The information about the signing certificate to be provided for a `describe-eif` request.
+#[derive(Clone, Serialize, Deserialize)]
+pub struct SignCertificateInfo {
+    #[serde(rename = "IssuerName")]
+    /// Certificate's subject name.
+    pub issuer_name: BTreeMap<String, String>,
+    #[serde(rename = "Algorithm")]
+    /// Certificate's signature algorithm
+    pub algorithm: String,
```

Comment:
can we also publish the certificates signature ?

Reply:
will add